extensions/wikia/Email/EmailController.class.php

```diff
@@ -28,6 +31,12 @@
 	/**	@var string The language to send the email in */
 	protected $targetLang;
 
```

Comment:
The following two members were pulled down from the WatchedPageController since they are common to all emails sent.